analysis/model/make_model_input.R

```diff
@@ -70,8 +71,18 @@ for (i in 1:nrow(active_analyses)) {
                            unlist(strsplit(active_analyses$covariate_other[i], split = ";")),#[!grepl("_priorhistory_",unlist(strsplit(active_analyses$covariate_other[i], split = ";")))],
                            "sub_cat_covid19_hospital",
                            "sub_bin_covid19_confirmed_history",
+                           "sub_bin_high_vascular_risk",
                            "cov_cat_sex",
                            "cov_num_age",
+                           "cov_bin_history_cog_imp_sympt",
+                           "cov_bin_history_alzheimer_disease",
+                           "cov_bin_history_vascular_dementia",
+                           "cov_bin_history_lewy_body_dementia",
+                           "cov_bin_history_any_dementia",
+                           "cov_bin_hypertension",
```

Comment:
`cov_bin_hypertension` and `cov_bin_diabetes` are not needed here as you defined the variable `sub_bin_high_vascular_risk`

Reply:
Done.